src/connectors/freemusicarchive.js

```diff
@@ -54,14 +54,15 @@ function setupCommonProps() {
 	Connector.pauseButtonSelector = '.playbtn-paused';
 
 	Connector.getUniqueID = () => {
-		const playingItem = document.querySelector('.gcol-electronic');
+		const playingItem = document.querySelector('.play-item.gcol-electronic');
 		if (!playingItem) {
 			return null;
 		}
 
 		const match = /(tid-\d+)/.exec(playingItem.className);
```

Comment:
If you really want to extract a numeric part, you can wrap `\d+` into a group.

Reply:
Will try that and update.